### DISCUSSION TITLE
'EngineIO' object has no attribute '_ping_loop'

### DISCUSSION THREAD

**Discussion by Sreenivasulu-Reddi** [miguelgrinberg/python-socketio#1218] [General]
currently iam upgrading the python libs in project ,iam getting this .can you pls, get me out of this.
latest versions:
python-socketio == 5.8.0
python-enginio == 4.5.1
websockets == 11.0.3
python ==3.11
client java socket.io =  4.7.1

**Comment by miguelgrinberg** [comment]
I need to see the complete stack trace please.
